### PR TITLE
feat: Change routes for RBAC

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/router.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ export const routes = [
 		},
 	},
 	{
-		path: '/workflows',
+		path: '/projects/:projectId/workflows',
```
Comment (mutdmour) [inline packages/editor-ui/src/router.ts:216]: Should not this be backwards compatible?

Comment (cstuncsik) [reply packages/editor-ui/src/router.ts:216]: We just agreed we would then keep the old routes and they'll redirect the users to the proper new ones

**Review by krynble — COMMENTED** [review]
One problem I saw while navigating the app is that if I go to `http://localhost:5678/executions` and click to `View` any execution it now fails.

It's worth going through all the app, as this might be happening in multiple places.

File: packages/editor-ui/src/features/projects/projects-routes.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+		],
+	},
+];
+
+export const projectsRouteBeforeMiddleware = async (
+	to: RouteLocationNormalized,
+	from: RouteLocationNormalized,
+	next: NavigationGuardNext,
+) => {
+	const projectId = 'home';
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/features/projects/projects-routes.ts:185]: That's another task when we will have the projects store

[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/features/projects/projects-routes.ts:185]: First I wanted to make sure new routes and redirections are working fine

**Review by valya — APPROVED** [review]

File: packages/editor-ui/src/components/WorkflowCard.vue [hunk 3]
```diff
@@ -221,7 +221,7 @@ export default defineComponent({
 				this.$telemetry.track('User opened sharing modal', {
 					workflow_id: this.data.id,
 					user_id_sharer: this.currentUser.id,
-					sub_view: this.$route.name === VIEWS.WORKFLOWS ? 'Workflows listing' : 'Workflow editor',
+					sub_view: 'Workflows listing',
```
Comment (krynble) [inline packages/editor-ui/src/components/WorkflowCard.vue:224]: We will have to circle back to this change, as we'll now have 3 possible places for the sharing modal:
- Home workflow list
- Project specific workflow list
- From within the canvas (while editing a workflow)

**Pull request merged**